core/base/base.go

```diff
@@ -145,14 +145,12 @@ func Start(swaggerFile string, registerHandlers bool) common.SyncServiceError {
 
 	if common.Configuration.NodeType == common.ESS {
 		common.Registered = false
-		if common.Configuration.CommunicationProtocol == common.HTTPProtocol {
-			go communication.Register()
```

Comment:
Why did you remove this?

Reply:
This was done to avoid 2 go funcs (this one and communications.ResendNotifications()) both calling Register since there wasn't anyway to guarantee which one ran first. This change makes it so only the ResendNotifications() method will handle the register for the ESS